### PR TITLE
Frontend ->  Master

### PR DESCRIPTION
**Pull request by kadirgokhann**
The latest and updated version of the frontend is now ready to be pushed to master.

### CODE REVIEW

**Comment by egetaga** [comment]
There are unresolved conflicts. You must solve them beforehand.

File: README.md [hunk 1]
```diff
@@ -66,32 +66,26 @@ expected to learn how to effectively use them
...
-
-### For Running Frontend App in Development Mode
+-> Checkout master branch
 
 -> Enter into the directory "frontend". 
 
--> npm i
-
--> npm run build
+-> Install dependencies with following commands:
```
Comment (hseyyidkaplan) [inline README.md:75]: dependency instructions updated by @hseyyidkaplan

File: deliverables/CMPE451_Customer_Presentation_Milestone_2/frontend/components/profile/Sidebar.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default function Sidebar({ menus }) {
           <Toolbar>
             <Link href="/homepage">
               <Image
-                src="/assets/BUdemi.svg"
+                src="/assets/budemi logo2.png"
```
Comment (hseyyidkaplan) [inline deliverables/CMPE451_Customer_Presentation_Milestone_2/frontend/components/profile/Sidebar.js:31]: logo updated by @mstfatay

File: frontend/.gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
-node_modules/*
\ No newline at end of file
+node_modules/*
```
Comment (hseyyidkaplan) [inline frontend/.gitignore:1]: gitignore updated by @hseyyidkaplan

File: frontend/components/detail/DetailMain.js [hunk 4]
```diff
@@ -1,97 +1,288 @@
 import * as React from "react";
-import { Grid, Box, Typography, Card, CardHeader, CardContent, Button } from "@mui/material";
-import Stack from '@mui/material/Stack';
-import InputLabel from '@mui/material/InputLabel';
-import MenuItem from '@mui/material/MenuItem';
-import FormControl from '@mui/material/FormControl';
-import Select, { SelectChangeEvent } from '@mui/material/Select';
+import {
```
Comment (hseyyidkaplan) [inline frontend/components/detail/DetailMain.js:2]: this page developed by @hseyyidkaplan and @kamilkorkut

**Review by hseyyidkaplan — COMMENTED** [review]
I completed my review and I am going to solve conflicts for merging.

**Review by omerozdemir1 — APPROVED** [review]
Let's merge this anyways. If it does not work, then we can destroy the front-end folder in master and merge front-end branch again.

**Pull request closed without merging**